lib/scrape.js

```diff
@@ -209,12 +215,24 @@ async function checkGitHubUserExists(user) {
 }
 
 async function searchGitHubOrgs(query) {
-  const res = await fetch(
-    `${GH_API_BASE}/search/users?q=${query}%20type:org`,
-    GH_API_OPTIONS
-  )
-  const { items } = await res.json()
-  return items || []
+  let data, errors
+  try {
+    ;({ data, errors } = await client.query(GITHUB_SEARCH_ORG_QUERY, { query }))
```

Comment:
Interesting, what is the semicolon for?

Reply:
Not sure, but `eslint` will be unhappy with that ;)

Reply:
Yes, this is because I put parentheses.
I have to use parentheses otherwise there would be a parsing error.